fix incorrect target in CMakeList (#38)
ZEISS/libczi: Src/libCZI/Doc/mainpage.markdown

For libCZI the [semantic versioning scheme 2.0](https://semver.org/) is to be applied. The version number is defined in the top-level CMakeList.txt file [here](https://github.com/ZEISS/libczi/blob/main/CMakeLists.txt#L4).
This is the only place where the version number is defined. When making changes, this version number needs to be updated accordingly.

> For determining whether a change is a breaking one, the source code level is decisive; i.e. not binary compatiblity (of the resulting static or dynamic libary) is to be considered, but the case

Reviewer: ```suggestion
For determining whether a change is a breaking one, the source code level is decisive; i.e. not binary compatibility (of the resulting static or dynamic library) is to be considered, but the case
```

Author: fixed - thanks and sorry

> of re-compiling the with the changed sources.

Reviewer: ```suggestion
of re-compiling with the changed sources.
```

> A borderline case here are maybe changes in the CMake-files - here the best judgment is to be applied whether a change in the compilation result will occur.

Reviewer: ```suggestion
A borderline case here are maybe changes in the CMake-files - here the best judgment is to be applied whether a change in the compilation result will occur.
```
```suggestion
A borderline case here are maybe changes in the CMake-files - here the best judgement is to be applied whether a change in the compilation result will occur.
```

Author: fixed - thanks and sorry
